docs/2023/Sunbird inQuiry/audio-upload-feature/updates/2023-07-28.md

```diff
@@ -4,13 +4,23 @@ author: Simran Nigam
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
-
+- [X] Audio Upload Enhancement Planning
+- [X] Audio upload enhancement Implementation
+- [X] Preparation and presention of Midpoint showcase 
 ## Screenshots / Videos 
+<img width="755" alt="17-06-22" src="https://github.com/Code4GovTech/c4gt-milestones/assets/95395832/3a9abdd7-a68d-4cd0-854e-22747ef006d6"/>
+<img width="755" alt="17-06-22" src="https://github.com/Code4GovTech/c4gt-milestones/assets/95395832/a83b6ee8-cd1e-4f8c-9132-c576adf4d55e"/>
 
 ## Contributions
+    https://github.com/Sunbird-inQuiry/editor/pull/75
+    
+    https://github.com/Sunbird-inQuiry/editor/pull/86
+
+## Learnings
+    Link to Midpoint Showcase presentation: https://docs.google.com/presentation/d/1z2F1amPsPX9BLjG-5cemsJwdmd-RWjU4Ecn8mK5eP3w/edit#slide=id.p1
```

Comment:
Instead of copying links create this section [titleForLink](link) This will create a simple anchor with the title and the redirection , instead of showing link as plain text